src/bentoml/serve.py

```diff
@@ -171,123 +173,6 @@ def find_triton_binary():
     return binary
 
 
-@inject
-def serve_http_development(
```

Comment:
Technically this is a breaking change if users are using this function. I think we should deprecate it until 1.1

I'm ok with making this breaking in 1.1

Reply:
is this a public api we document?

Reply:
we don't, but i believe there are users that use this internal API